src/core/db/manager.py

```diff
@@ -259,13 +301,22 @@ def list_collaborators(self, repo):
         # [{'username': 'foo_user', 'permissions': 'UC'},
            {'username': 'bar_user', 'permissions': 'U'}]
         """
+        # get the database's idea of permissions
         with _superuser_connection(self.repo_base) as conn:
-            result = conn.list_collaborators(repo=repo)
-        return result
+            db_collabs = conn.list_collaborators(repo=repo)
+
+        # merge it with the datahub collaborator model permissions
+        dh_collabs = Collaborator.objects.all()
```

Comment:
Thank you! updated in 3e91d744b84c5f623e4ff5a65a3d82ebdd7cd0bf. Setting to `''` is the desired behavior.